.github/workflows/lighthouse-ci.yml

```
@@ -0,0 +1,14 @@
+name: CI
+on: [push]
+jobs:
+  lighthouseci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

We can eliminate a magic string here by following the guidance of the existing workflow file at https://github.com/nodejs/nodejs.org/blob/main/.github/workflows/pull-request.yml

```suggestion
          node-version-file: '.nvmrc'
```

---

Done